### PR TITLE
[WIP] Notification Drawer changes required for PatternFly latest packages

### PR DESCRIPTION
**Pull request by dtaylor113**
Notification Drawer is labeled `beta` in PF meaning breaking changes can occur.
This PR updates OpenShift's usage of NotificationDrawer to adhere to latest PF implementation.

TODO:
- rebase once #4704 has been merged.

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
/approve

can swing back around for further review when WIP gone.

File: frontend/packages/patternfly/src/components/notification-drawer/notification-drawer.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import {
   Drawer,
   DrawerContent,
   DrawerPanelContent,
+  DrawerPanelBody,
 } from '@patternfly/react-core/dist/js/experimental/components/Drawer';
```
Comment (dtaylor113) [inline frontend/packages/patternfly/src/components/notification-drawer/notification-drawer.tsx]: TODO: I thought patternfly switched from 'experimental' to 'beta'.  We probably need to change this import before merge!

**Comment by spadgett** [comment]
>     * rebase once #4704 has been merged.

Let's roll this into one PR with all the changes needed for the PF upgrade

**Comment by dtaylor113** [comment]
Closing in favor of #5078

**Pull request closed without merging**